cmd/convox/start.go

```diff
@@ -42,6 +44,14 @@ func cmdStart(c *cli.Context) {
 		stdcli.QOSEventSend("cli-start", distinctId, stdcli.QOSEventProperties{Error: err})
 	}
 
+	dockerTest := exec.Command("docker", "images")
+	err = dockerTest.Run()
+	if err != nil {
+		stdcli.QOSEventSend("cli-start", distinctId, ep)
```

Comment:
this is not tracked as an exception, just a metric, the ep var has no error in it